app/components/ChallengeDetailComponents/NumberCompleteView.js

```diff
@@ -16,11 +16,11 @@ export default class NumberCompleteView extends Component {
     return (
     <View style = {styles.numcompletedarea}>
       <Image
-        style = {{width: 25, height: 25}}
+        style = {{width: 25, height: 25, marginLeft: 12}}
         source = {accountmultiple}/>
-      <Text>     Completed by </Text>
+      <Text style = {{paddingLeft: 24}}>Completed by </Text>
       <Text>{this.props.challenge.completedBy.length}</Text>
-      <Text> people.</Text>
+      <Text> people</Text>
```

Comment:
I know this isn't styling but I would fix this now as well. You're using multiple `<Text>` tags here and don't need to be. Instead of having:
```javascript
<Text> Completed by</Text>
<Text>{this.props.challenge.completedBy.length} </Text>
<Text> people.</Text>
```
You can squish this all down to:
```javascript
<Text> Completed by {this.props.challenge.completedBy.length} people.</Text>
```

The curly brackets tell react to evaluate the value inside rather than just printing out that string. Adding those extra `<Text>` tags only muddies the code. The only reason you would want them is if you were wanting to style each text block differently.